### DISCUSSION TITLE
Novità

### DISCUSSION THREAD

**Discussion by martinapavonidoc1** [iisgiua/giuaschool#678] [Idee e proposte]
Buonasera, ci sono novità per la nuova versione ? cosa verrà implementato?

**Comment by trinko** [comment]
Nella versione 1.5.8, per ora è previsto:
- nuova gestione note disciplinari, con annullamento formale, separazione tra nota e provvedimento disciplinare;
- correzione e modifiche allo scrutinio finale, con inserimento della nuova certificazione delle competenze;
- gestione RSPP e possibilità di inserirlo come destinatario degli avvisi;
- correzioni di vari bug minori.

**Reply by Davide031** [reply]
Si sa già quando uscirà?

**Comment by trinko** [comment]
> Si sa già quando uscirà?

Penso entro la fine di Maggio
